### ISSUE TITLE
Bakery doesn't run `MIDDLEWARE_CLASSES`/`MIDDLEWARES` on build

### ISSUE THREAD

**Issue by Rafalonso** [palewire/django-bakery#132]
How can I make this happen? I'm using `BuildableTemplateView`.

Django 1.8
django-bakery 0.12.6

**Comment by palewire** [comment]
Can you tell me more about your bug?

**Comment by Rafalonso** [comment]
I don't believe this is a bug but a feature. Even though it does run `TEMPLATE_CONTEXT_PROCESSORS` as desired, it does not run `MIDDLEWARES`. Makes sense since middlewares handle dynamic requests. Would be nice though to force bakery to run them with a flag.

**Comment by palewire** [comment]
What's the use case for needing the middleware?

**Comment by palewire** [comment]
django-bakery builds pages from synthetic `GET` requests via `RequestFactory`, not through Django's full HTTP request handler, so Django middleware is intentionally not executed during static builds.

For build-specific request data, override `BuildableMixin.create_request()`; see the current API docstring and the [Static build requests documentation](https://django-bakery.readthedocs.io/en/latest/buildableviews.html#static-build-requests). This is the supported extension point, but middleware execution itself is not supported for static builds.
